README.md

```diff
@@ -1,5 +1,5 @@
-About rubin-env
-===============
+About rubinenv
```

Comment:
It's unfortunate that this page becomes more confusing with the multiple outputs and dash vs. non-dash.  Since I think this is auto-rendered, I guess there's not much that can be done about it.

Reply:
Yep. If you leave the name as is in the recipe, conda-build gets confused a lot.